cli_tools_tests/infra/no-external-ip/README

```diff
@@ -0,0 +1,9 @@
+Creates a GCP project that disables external IPs for VMs. Internet traffic is routed through
```

Comment:
it's not only no-external-ip but also no default network

Reply:
Updated to clarify that the default network is removed.